Invoro.WebApp/Invoro/invoro/src/lib/components/Vote/Vote.component.test.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { mount, ReactWrapper } from 'enzyme';
+import VoteComponent from './Vote.component';
+
+describe("VoteComponent", () => {
+    let voteComponent: ReactWrapper;
+
+    describe("Star Icon rendered when isVoted is true", () => {
```

Comment:
I would add another describe here:
describe("Star Icon rendered)
    describe("When is isVoted is true)
        it("")
        it("")
    describe("When is isVoted is false)
        it("")
        it("")